cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -129,19 +129,21 @@ private void loadMutationRecordsFromJson(CVRData cvrData) {
         List<MutationRecord> recordsToAnnotate = new ArrayList<>();
         for (CVRMergedResult result : cvrData.getResults()) {
             String sampleId = result.getMetaData().getDmpSampleId();
-            String somaticStatus = result.getMetaData().getSomaticStatus() != null ? result.getMetaData().getSomaticStatus() : "N/A";
             int countSignedOutSnps = result.getAllSignedoutCvrSnps().size();
-            for (CVRSnp snp : result.getAllSignedoutCvrSnps()) {
-                MutationRecord to_add = cvrUtilities.buildCVRMutationRecord(snp, sampleId, somaticStatus);
-                recordsToAnnotate.add(to_add);
-                addRecordToMap(to_add);
-            }
             cvrSampleListUtil.updateSignedoutSampleSnpCounts(sampleId, countSignedOutSnps);
+	    if (cvrSampleListUtil.getPortalSamples().contains(sampleId)) {
```

Comment:
Samples had been removed if they were not in `cvrSampleListUtil.getPortalSamples()` after being annotated.  I decided to remove them right away instead.  This speeded things up considerably.

Reply:
@n1zea144 and I just had a chat about this and since updateSignedoutSampleSnpCounts() was previously called no matter what we are still going to do that (even though I also am not quite sure why this is the case).